app/file.go

```diff
@@ -799,26 +800,25 @@ func (t *UploadFileTask) postprocessImage() {
 	writeJPEG := func(img image.Image, path string) {
 		r, w := io.Pipe()
 		go func() {
-			_, aerr := t.writeFile(r, path)
-			if aerr != nil {
-				mlog.Error("Unable to upload", mlog.String("path", path), mlog.Err(aerr))
-				return
+			err := jpeg.Encode(w, img, &jpeg.Options{Quality: 90})
```

Comment:
Can you make Quality value into some default constant and add a comment to it?

Reply:
done. And I also reverted the extra checks that were a TODO statement, which I forgot to remove.